config/karton.docker.ini

```diff
@@ -1,15 +1,14 @@
 [redis]
-host=localhost
-port=8379
+host=redis
 
 [minio]
-secret_key = mwdbmwdb
 access_key = mwdb
-address = localhost:8090
+secret_key = mwdbmwdb
+address = minio:9000
 bucket = karton
 secure = 0
 
 [mwdb]
-api_url=http://localhost:8080/api/
+api_url = http://mwdb.:8080/api/
```

Comment:
> `mwdb.`

Is the dot here on purpose?

Reply:
Yes, it indicates a fully qualified domain name (no DNS suffixes applied)